components/filter/filter.js

```diff
@@ -487,15 +494,26 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 			if (count === 0) return searchResults;
 		}
 
+		let listHeader = nothing;
+		if (dimension.headerText && dimension.searchValue === '') {
+			listHeader = html`
+				<d2l-list-item>
+					<h4 class="d2l-heading-4 list-header-text" aria-hidden="true">${dimension.headerText}</h4>
+				</d2l-list-item>
+			`;
+		}
+
 		return html`
 			${searchResults}
 			<d2l-list
 				id="${SET_DIMENSION_ID_PREFIX}${dimension.key}"
 				@d2l-list-selection-change="${this._handleChangeSetDimension}"
 				extend-separators
 				grid
+				label="${ifDefined(dimension.headerText)}"
```

Comment:
Kinda weird, but to be accurate to Wave's case, should we _not_ set this if a search has been done (like we do for the visual header)?  Since the search results might _not_ be promoted by the company?

Reply:
Yeah, you are correct. It's a bit weird but the label should not be there when searching